cpp/src/gandiva/engine.cc

```diff
@@ -66,6 +66,10 @@
 
 namespace gandiva {
 
+GANDIVA_EXPORT
```

Comment:
Why the export? I'm expecting the bitcode object file to be linked statically with this, and never referenced from the outside.

Reply:
It was used for the bitcode path previously. Removing.